### PR TITLE
[18269] Documentation: CLI verb set

### PR DESCRIPTION
**Pull request by JesusPoderoso**

### CODE REVIEW

File: docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst [hunk 1]
```diff
@@ -2,3 +2,68 @@
 
 Fast DDS QoS Profiles Manager CLI ``set`` verb
 ----------------------------------------------
+
+Passing to the CLI the verb ``set`` as the third argument would include the given ``value`` in the selected XML ``element``.
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst:6]: I suggest giving the information split for better consumption hehehe

```suggestion
Passing to the CLI the verb ``set`` would include the given ``value`` in the selected XML ``element``.
The verb is the third argument passed to the CLI (reference to the usage section).
The element is given by a dot-separated string accessing the configuration parameter to be set (reference to the Configuration parameter access section).
The set command requires usually at least the value desired for the specified element.
(Here the note stating the exception to the one value can be)
(Once complex commands are supported, this section will need to be revisited).
```

File: docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst [hunk 2]
```diff
@@ -2,3 +2,68 @@
...
+
+.. note::
+
+    There are exceptions in which the verb ``set`` does not need a ``value``, as in ``default_profile`` cases for the entities ``datareader``, ``datawriter`` and ``participant``.
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst:10]:
```suggestion
    There are exceptions in which the verb ``set`` does not need a ``value``, as in the ``default_profile`` attribute cases for ``datareader``, ``datawriter`` and ``participant`` entities.
```

File: docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst [hunk 3]
```diff
@@ -2,3 +2,68 @@
...
+                </rtps>
+            </participant>
+            <transport_descriptors>
+                <transport_descriptor>
+                    <transport_id>td_profile</transport_id>
+                    <type>SHM</type>
+                </transport_descriptor>
+            </transport_descriptors>
+        </profiles>
+    </dds>
```
Comment (JLBuenoLopez-eProsima) [inline docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst:69]: Let us follow the same criteria as in Fast DDS documentation. Every XML snippet should be in another file and included here from the file.

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]

File: docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst (hunk 1 above)
Comment (JesusPoderoso) [reply docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst:6]: Notes were taken to revisit this section when complex commands supported

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Pull request merged**